src/errors/Messages.js

```diff
@@ -17,6 +17,8 @@ const Messages = {
 
   SHARDING_INVALID: 'Invalid shard settings were provided.',
   SHARDING_REQUIRED: 'This session would have handled too many guilds - Sharding is required.',
+  INVALID_INTENTS: 'There is an invalid intent for websocket intents.',
+  DISALLOWED_INTENTS: 'You included an intent on your websocket intents which you do not have privilege on.',
```

Comment:
"You included a WebSocket intent that you do not have permissions to use."

Also, would we guide people to the dev portal to fix it? Or leave them be?

Reply:
I think its fine to add the link to the actual error messages, if you don't mind it long

Reply:
```suggestion
  DISALLOWED_INTENTS: 'Privileged intent provided is not enabled or whitelisted.',
```

Reply:
I rephrased both the error messages on what Crawl's suggested